docs/src/app/components/pages/components/lists.jsx

```diff
@@ -35,10 +35,10 @@ let CodeExample = require('../../code-example/code-example');
 
 
 
-class ListsPage extends React.Component {
+export default class ListsPage extends React.Component {
 
-  constructor() {
-    super();
+  constructor(props) {
```

Comment:
@NogsMPLS I have never use ES6 Class definition. I was just considering the git diff. Ok make sense :+1: